frontend/src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,40 @@
+import AvatarCount from './AvatarCount';
+import AvatarItem from './AvatarItem';
+import { avatarContainerStyle } from './index.css';
+
+export type Size = 'sm' | 'lg';
+
+interface AvatarProps {
+  size: Size;
+  imageUrls: string[];
+}
+
+const MAX_IMAGE_COUNT = 4;
+
+const Avatar = ({ size, imageUrls: prevUrls }: AvatarProps) => {
+  let limitedUrls = prevUrls;
+  if (prevUrls.length > MAX_IMAGE_COUNT) {
+    limitedUrls = prevUrls.slice(0, MAX_IMAGE_COUNT);
+  }
```

Comment:
```suggestion
const ENTIRE_LENGTH = imageUrls.length;
const limitedUrls = imageUrls.slice(0, MAX_IMAGE_COUNT);
```
p2; 
불필요한 `let` 사용인 것 같습니다. 위와 같이 수정하면 어떨까요? 배열보다 큰 범위여도 원본 배열의 내용이 리턴될 것 같아요.
이후에 리턴문의 조건문에서는 `ENTIRE_LENGTH > MAX_IMAGE_COUNT`으로 쓸 수도 있고요!

Reply:
더 깔끔하네요 적용해보겠습니다!